src/components/MultiSelect/FilterableMultiSelect.js

```diff
@@ -239,36 +285,98 @@ export default class FilterableMultiSelect extends React.Component {
                   <ListBox.MenuIcon isOpen={isOpen} />
                 </ListBox.Field>
                 {isOpen && (
-                  <ListBox.Menu>
-                    {sortItems(
-                      filterItems(items, { itemToString, inputValue }),
-                      {
-                        selectedItems,
-                        itemToString,
-                        compareItems,
-                        locale,
-                      }
-                    ).map((item, index) => {
-                      const itemProps = getItemProps({ item });
-                      const itemText = itemToString(item);
-                      const isChecked =
-                        selectedItem.filter(selected => isEqual(selected, item))
-                          .length > 0;
+                  <ListBox.Menu style={wrapper}>
+                    {groupedByCategory(items).map(group => {
+                      const categoryName = group[0].toUpperCase();
```

Comment:
If `group[0]` is undefined this will throw an error. I think the next line should be above it and a check before trying to pull the value. Will it always be the 0th element?

Reply:
Will make that validation, it makes sense thanks!. And yes the structure of the array returned by the groupedByCategory(items) function is the follow 
` [["Category 1", Array(1)],["Category 2", Array(1)],["Category 3", Array(1)]`
So every 0th position of the resulted array will be the name of the category their corresponded items belong to.